roadmap.md

```diff
@@ -9,12 +9,12 @@ the Ellaism blockchain. Ellaism Core is a non-profit team that develops and
 maintains some core infrastructures on the Ellaism blockchain.
 
 You can donate to Ellaism Core by supporting the **Dev Fund**, which you can
-find more information on [Donate to Ellaism](/donations/) page.
+find more information on at the [Donate to Ellaism](/donations/) page.
 
 ## Funding and Community Efforts
 
-One of the most important things that users would care about would be how
-Ellaism Core is funded and how the team would impact the community. Here we make
+One of the most important things that users should care about should be how
```

Comment:
Originally this means "how Ellaism Core team impact the community" is something that users **may** want to know about first. So I think it's better to change to "One of the most important things that users care about *is* ..."?